gbpservice/nfp/orchestrator/coal/networking/nfp_gbp_network_driver.py

```diff
@@ -59,3 +65,8 @@ def set_promiscuos_mode(self, token, port_id):
         self.network_handler = openstack_driver.NeutronClient(self.config)
         super(NFPGBPNetworkDriver, self).set_promiscuos_mode(token, port_id)
         self.network_handler = openstack_driver.GBPClient(self.config)
+
+    def set_promiscuos_mode_v1(self, token, port_id):
```

Comment:
Check complete code plz, this is invoked in a separate thread paralell to create_interface and also optimizations done, compare it w.r.t existing set_promiscuous_mode to see the difference. Please follow the same for all further function with _v1